src/components/index.md.njk

```diff
@@ -6,6 +6,8 @@ show_subnav: true
 
 Components are reusable parts of the user interface that have been made to support a variety of applications.
 
-Individual components can be used in multiple different [patterns](../patterns) and contexts. For example, the [text input](../components/text-input) component can be used to ask for an email address, a National Insurance number or someone’s name.
+You can use individual components in many different [patterns](../patterns) and contexts. For example, you can use the [text input](../components/text-input) component to ask for an email address, a National Insurance number or someone’s name.
 
-If you are using the [GOV.UK Prototype Kit](https://govuk-prototype-kit.herokuapp.com) or have [GOV.UK Frontend](https://frontend.design-system.service.gov.uk/) included in your build, the coded examples provided will render exactly as they do inside the Design&nbsp;System.
+If you are using the [GOV.UK Prototype Kit](https://govuk-prototype-kit.herokuapp.com), or have included [GOV.UK Frontend](https://frontend.design-system.service.gov.uk/) in your build, the coded examples provided will render exactly as they do inside the GOV.UK Design System.
+
+The GOV.UK Design System's code is public and freely available under an MIT license. You can [find our code repositories on GitHub](https://github.com/alphagov?q=design-system-), where we [code in the open](https://gds.blog.gov.uk/2012/10/12/coding-in-the-open/).
```

Comment:
@EoinShaughnessy Do you think we need to make it clear that the MIT license applies to both the code behind the GOV.UK Design System *and* GOV.UK Frontend? In the sentence above, we refer to the 'GOV.UK Design System' as the website which displays these components, patterns etc - whereas I think here we're using GOV.UK Design System to mean GOV.UK Frontend + Prototype Kit + the design system website

Reply:
@vanitabarrett Really good shout! How does this look?

```suggestion
The code for both the GOV.UK Design System and GOV.UK Frontend is public and freely available under an MIT license. You can [find our code repositories on GitHub](https://github.com/alphagov?q=design-system-), where we [code in the open](https://gds.blog.gov.uk/2012/10/12/coding-in-the-open/).
```

Reply:
@EoinShaughnessy I wonder if we should link to one of the license files on Github as it gives more details on what an MIT license actually means? E.g: https://github.com/alphagov/govuk-frontend/blob/main/LICENSE.txt

Also, I wonder if we can link to the [govuk-design-system-team tag](https://github.com/topics/govuk-design-system-team) on Github, it seems to work a bit better than the search query which excludes things like `govuk-frontend`.

Reply:
Thanks, those suggestions both make sense. How's this?

```suggestion
The GOV.UK Design System's code is public and [freely available under an MIT license](https://github.com/alphagov/govuk-frontend/blob/main/LICENSE.txt). You can [find our code repositories on GitHub](https://github.com/alphagov?q=design-system-), where we [code in the open](https://github.com/topics/govuk-design-system-team).
```

Reply:
@EoinShaughnessy Looks good to me!

Reply:
@vanitabarrett Cheers! I'll send this update for pre-i.